Gemfile

```diff
@@ -16,7 +16,9 @@ gemspec
 gem 'schema_validations',  git: 'https://github.com/SchemaPlus/schema_validations.git'
 
 group :test do
-  gem 'capybara-webkit'
+  gem 'capybara'
+  gem 'capybara-selenium'
```

Comment:
What's driving these changes?

Reply:
capybara-webkit is no longer supported and can't be installed on newer version of macOS (like the one I'm running). I was unable to work on the gem without making these changes. This now uses the same gems for browser tests that tracker does

Reply:
Ok, I wasn't aware of that. It's working for the rest of us on our dev machines, but we'll start the process of switching over to the new gems. I do remember it being a pain to install the native libraries. I might pull this out separately ahead of time depending on timing.

Reply:
@elbzero see above